tests/api/fixtures/users.js

```diff
@@ -33,7 +33,7 @@ const API = module.exports = {
     return signup({
       email,
       username: API.createUsername(),
-      password: faker.internet.password()
+      password: randomBytes(8)
```

Comment:
`randomBytes` returns a buffer object
```suggestion
      password: randomBytes(8).toString('base64')
```

Reply:
done in 3c173fedd